src/content/developers/docs/gas/index.md

```diff
@@ -36,11 +36,50 @@ Although a transaction includes a limit, any gas not used in a transaction is re
 ![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
+## Why can gas fees get so high? {#Why-can-gas-fees-get-so-high}
+
+High gas fees are due to the popularity of Ethereum. To perform any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.
```

Comment:
"To perform any operation on Ethereum requires consuming gas." The tense feels off here... maybe:
"_Performing_ any operation on Ethereum requires consuming gas."